test/model/unreads_test.dart

```diff
@@ -174,7 +174,7 @@ void main() {
       check(model.countInCombinedFeedNarrow()).equals(5);
     });
 
-    test('countInStream/Narrow', () async {
+    test('countInChannel/Narrow', () async {
```

Comment:
This change doesn't seem right — it's referring to a pair of identifiers, and…

oh I see, one of them changed and the other didn't.

These two method names are meant to parallel each other, because their meanings are so closely related. I'll add a followup commit renaming the other method.